Assets/MixedRealityToolkit.Extensions/Sharing/Localization/Extensions.cs

```diff
@@ -0,0 +1,140 @@
+
+        /// <summary>
+        /// Converst coordinate space position to world space position.
+        /// </summary>
+        public static UVector3 CoordinateToWorldSpace(this ISpatialCoordinate coordinate, UVector3 vector) => coordinate.CoordinateToWorldSpace(vector.AsNumericsVector()).AsUnityVector();
+
+        /// <summary>
+        /// Converst coordinate space position to world space position.
+        /// </summary>
+        public static UQuaternion CoordinateToWorldSpace(this ISpatialCoordinate coordinate, UQuaternion quaternion) => coordinate.CoordinateToWorldSpace(quaternion.AsNumericsQuaternion()).AsUnityQuaternion();
+
+        /// <summary>
+        /// Attempts to create a new coordinate with this service.
+        /// </summary>
+        /// <param name="localPosition">Position at which the coordinate should be created.</param>
+        /// <param name="localRotation">Orientation the coordinate should be created with.</param>
+        /// <returns>The coordinate if the coordinate was succesfully created, otherwise null.</returns>
+        public static Task<ISpatialCoordinate> TryCreateCoordinateAsync(this ISpatialCoordinateService spatialCoordinateService, UVector3 vector, UQuaternion quaternion, CancellationToken cancellationToken)
+            => spatialCoordinateService.TryCreateCoordinateAsync(vector.AsNumericsVector(), quaternion.AsNumericsQuaternion(), cancellationToken);
+
+        /// <summary>
+        /// Gracefully allows a task to continue running without loosing any exceptions thrown or requireing to await it.
+        /// </summary>
+        /// <param name="task">The task that should be wrapped.</param>
+        public static async void FireAndForget(this Task task)
+        {
+            try
+            {
+                await task.IgnoreCancellation().ConfigureAwait(false);
+            }
+            catch (Exception ex)
+            {
+                UnityEngine.Debug.LogError("Encountered an exception with a FireAndForget task.");
+                UnityEngine.Debug.LogException(ex);
+            }
+        }
+
+        /// <summary>
+        /// Gracefully allows a task to continue running without loosing any exceptions thrown or requireing to await it.
+        /// </summary>
+        /// <param name="task">The task that should be wrapped.</param>
+        public static async void FireAndForget<T>(this Task<T> task)
+        {
+            try
+            {
+                await task.IgnoreCancellation().ConfigureAwait(false);
+            }
+            catch (Exception ex)
+            {
+                UnityEngine.Debug.LogError("Encountered an exception with a FireAndForget task.");
+                UnityEngine.Debug.LogException(ex);
+            }
+        }
+
+        /// <summary>
+        /// Prevents <see cref="TaskCanceledException"/> or <see cref="OperationCanceledException"/> from trickling up.
```

Comment:
>OperationCanceledException [](start = 71, length = 26)

nit: I'm personally curious whether these exceptions resolve correctly with docfx.

Reply:
They should be able to, as VS intellisense resolved them.

---
In reply to: [270515114](https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/3732#discussion_r270515114) [](ancestors = 270515114)

Reply:
I know that Bernadette stated that there may be issues with resolving namespaces with docfx. I know i have hit issues where editor items couldn't resolve classes across asmdefs correctly. It's not a big deal for now but may break the build when opening a pr against mrtk_development